src/lib/stores/campaigns.ts

```diff
@@ -68,6 +68,26 @@ campaigns
             ]
         }
     })
+    .set('Free100', {
```

Comment:
If im not wrong we also need to add images for the campaign landing page when the user is not logged in on the console like so https://github.com/appwrite/console/pull/1267/files

Please check the scenario when you visit the console ( cloud.appwrite.io?code=<code> ) without a session .. And it should take you to a campaign landing page. Lets make sure there is an image present there

Reply:
As this is review template, this has only review image which is added
https://github.com/appwrite/console/pull/1277/files#diff-273f15cb5da459cf45dfbf2bd22b50fda7b488177e304c25bac854f1e6a2201a